apps/src/studio/studio.js

```diff
@@ -1711,11 +1712,15 @@ Studio.initSprites = function () {
         if (0 === Studio.spriteCount) {
           Studio.spriteStart_ = [];
         }
-        Studio.spriteStart_[Studio.spriteCount] = Object.assign({},
-            Studio.map[row][col].serialize(), {
+        var cell = Studio.map[row][col].serialize();
+        Studio.spriteStart_[Studio.spriteCount] = Object.assign({}, cell, {
               x: col * Studio.SQUARE_SIZE,
               y: row * Studio.SQUARE_SIZE
             });
+        if (cell.sprite !== undefined) {
+          presentAvatars[Studio.spriteCount] =
+            skin.avatarList[cell.sprite + level.firstSpriteIndex];
```

Comment:
wouldn't we just want this to be `cell.sprite`? I thought the assigned sprites did not respect the offset